plugins/profefe.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: profefe
+spec:
+  version: v0.8.0
+  homepage: https://github.com/profefe/kube-profefe
+  shortDescription: continuous profiling made easy in Kubernetes with profefe
+  description: |
+    This plugin helps you do manage pprof profiles via Profefe. It helps you do
+    grab profiles from running pod. Load profile you have locally in Profefe
+    and inspect the profiles you have stored.
+  platforms:
+  - selector:
+      matchLabels:
+        os: darwin
+        arch: amd64
+    uri: "https://github.com/profefe/kube-profefe/releases/download/v0.0.8/kube-profefe_0.0.8_Darwin_x86_64.tar.gz"
+    sha256: 4c0ce3fd50b053ae91f697099ab7cc6abfa1b7d79b326c64987933fd1814844c
+    files:
+    - from: "*"
+      to: "."
```

Comment:
this is the default, you can omit `files:` or just type `files:` with no content.

also please make sure there's a `LICENSE` file extracted with this.